test_lib/compaction.py

```diff
@@ -23,6 +38,40 @@ def from_str(cls, output_str):
             raise ValueError(err_msg) from attr_err
 
 
+def get_gc_mode(node, keyspace, table) -> None | GcMode:
+    """Get a given table GC mode
+
+    :Arguments:
+        node {str} -- ip of db_node
+        keyspace
+        table
+    """
+    list_tables_gc_mode = node.run_cqlsh('SELECT keyspace_name, table_name, extensions FROM system_schema.tables',
```

Comment:
can we use query like:
```
'SELECT extensions FROM system_schema.tables where keyspace_name = '{keyspace}' and table = '{table}'?
```
then following logic could be simpler

Reply:
fixed